lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -574,6 +587,27 @@ migrateManually tr proxy defaultFieldValues =
         renameColumn conn (DBField SeqStateAddressAccountingStyle)
             "u_tx_o_external" "utxo_external"
 
+    updateFeeValue :: Sqlite.Connection -> IO ()
+    updateFeeValue conn = do
+        fee_policyInfo <- Sqlite.prepare conn $ T.unwords
+            [ "SELECT fee_policy FROM protocol_parameters"
+            , ";"
+            ]
+        row <- Sqlite.step fee_policyInfo
+            >> Sqlite.columns fee_policyInfo
+        Sqlite.finalize fee_policyInfo
+        case row of
+            [PersistText t] -> do
+                (a:b:_) <- pure (T.splitOn " + " t)
+                let newVal = a <> " + " <> b
+                query <- Sqlite.prepare conn $ T.unwords
+                    [ "UPDATE protocol_parameters"
+                    , "SET fee_policy", "=", newVal
+                    , ";"
+                    ]
+                Sqlite.step query *> Sqlite.finalize query
+            _ -> pure ()
```

Comment:
Please stick to the existing patterns with regards to database migrations: 

a) Avoid hard-coded column and table names, and use instead the `DBField` data-type for existing database types.

b) Define proper logging messages depending on the situation (migration needed vs migration not needed)

c) Make sure to add a unit test for testing the migration (see examples in https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/test/unit/Cardano/Wallet/DB/SqliteSpec.hs#L250-L272) 

d) Make sure to start your migration query with an `isFieldPresent` request. The database may be older than you think.